### ISSUE TITLE
Ctrl-tab

### ISSUE THREAD

**Issue by madeso** [madeso/ride#43]
Implement a ctrl-tab dialog like visual studio to easily switch between documents

**Comment by madeso** [comment]
http://www.kirix.com/forums/viewtopic.php?f=16&t=528&start=0&st=0&sk=t&sd=a

> Hi,
> 
> As prompted by a bug report about the lack of keyboard navigation in wxAUI, here's a class that can help a bit (though we should still strive for 'native' navigation through all the windows in an app).
> 
> Screenshot:
> http://www.anthemion.co.uk/wxwin/switch ... indows.jpg
> 
> Code and Windows binary of the wxAUI demo app (press Ctrl+Tab):
> http://www.anthemion.co.uk/wxwin/switcher.zip
> 
> Readme:
> http://www.anthemion.co.uk/wxwin/readme.txt
> 
> I'll include the readme below. It's a general-purpose class but can very easily be used for wxAUI applications. Perhaps it's something that we should think of including in wxWidgets 2.9/3 if we don't have another built-in method of navigating around wxAUI panes.
> 
> BTW you can also see this class in action in DialogBlocks 4.15 (Windows, Linux, Solaris, FreeBSD and Mac).
> 
> Regards,
> 
> Julian

**Comment by madeso** [comment]
This is pretty much done. It only needs another color setting for the highlighted text.
